get.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require "connect.php";
+
+function get($db) {
+
+$query = $db->prepare("SELECT `picture`, `species`, `color`, `frequency`, `shit` FROM `birds`;");
+
+$query->setFetchMode(PDO::FETCH_ASSOC);
```

Comment:
If you're always going to use fetch_assoc (imagine you are!) then it might be worth setting it on the $db in connect.php instead of here, that way you can just set it straight away and forget about t.

Reply:
cant remember the code for this and google is not helping